kratos/includes/define.h

```diff
@@ -810,6 +810,36 @@ namespace Kratos
 //Print Trace if defined
 #define KRATOS_WATCH(variable) std::cout << #variable << " : " << variable << std::endl;
 
+// Print vector values with precision
+#define KRATOS_WATCH_VECTOR_WITH_PRECISION(variable, precision)                           \
+    {                                                                                     \
+        std::stringstream values;                                                         \
+        values << std::scientific << std::setprecision(precision);                        \
+        for (std::size_t i = 0; i < variable.size(); i++)                                 \
```

Comment:
why doing it manually and not using the stl_vector_io?

Reply:
i need the index of the position as well as the value. Is there a way to get both cleanly with stl_vector_io?